sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat_metrics.py

```diff
@@ -215,6 +219,10 @@ def _get_feature_metric(self, options: CallbackOptions) -> Iterable[Observation]
         if not self._meets_long_interval_threshold(_FEATURE_METRIC_NAME[0]):
             return observations
         # Feature metric
+        # Check if any features were enabled during runtime
+        if get_statsbeat_custom_events_feature_set():
+            self._feature |= _StatsbeatFeature.CUSTOM_EVENTS_EXTENSION
+            _StatsbeatMetrics._FEATURE_ATTRIBUTES["feature"] = self._feature
```

Comment:
Are we currently only checking for updates on custom events?

Reply:
I will add these in a different pr if needed.